### DISCUSSION TITLE
Implementing copilot in my custom editor.

### DISCUSSION THREAD

**Discussion by vrunhofen** [community/community#20520] [Copilot Conversations]
I have an active subscription to copilot and it works well using VS Code.
I also have a custom editor that I built for visual programming. It uses electron js and for the editor itself CodeMirror. 
I would like to use copilot in this custom editor. And I've searched repos, and gone through a bunch of code, but I can't figure out how to add copilot to my editor. Are you guys looking into this? I mean unless you are using the 4 supported editors, you can't use copilot. But copilot is a general tool, and it should be trivial to implement it in my own editor if there is an API that is exposed and documented. 
Please suggest ways I can integrate copilot into my custom editor.

**Comment by D7EAD** [comment]
Copilot, as of right now, is only available to a specific set of editors and IDEs. It is unclear if this will change in the future, but hopefully it does.
